defaults/main.yml

```
@@ -5,12 +5,15 @@ ethv_ssh_port: "22"
 # Number of validators
 ethv_number_validators: "1"
 
-# The http endpoint for an Ethereum 1.0 node (e.g. infura)
-ethv_10_http_provider: "http://127.0.0.1:8545"
-
 # Geth cache
 ethv_10_cache: "1024"
 
+# The http endpoint for an Ethereum 1.0 node (e.g. infura), this is for prysm only (below is for lighthouse)
+ethv_10_http_provider: "http://127.0.0.1:8545"
```

```suggestion
ethv_10_prysm_http_provider: "http://127.0.0.1:8545"
```

Maybe we could differentiate with the variable name here, to make it more clear?

---

've heard prysm are adding support for fallback web3 endpoints. Once this is done we will be able to use ethv_10_http_providers and remove ethv_10_http_provider entirely so I don't know if it's cleaner to leave them as is.

---

It is definitely confusing as it is but I'm trying to keep API changes to a minimum in case others are already using this role.